README.md

```diff
@@ -705,6 +705,7 @@ Table of Contents
    * [Quickmetrics](https://www.quickmetrics.io/) — Timeseries database with dashboard included, free up to 10,000 events/day and total of 5 metrics.
    * [restdb.io](https://restdb.io/) - a fast and simple NoSQL cloud database service. With restdb.io you get schema, relations, automatic REST API (with MongoDB-like queries) and an efficient multi-user admin UI for working with data. Free plan allows 3 users, 2500 records and 1 API requests per second.
    * [Unbounded](https://www.unbounded.cloud/) - a NoSQL DBaaS which uses Javascript as a query language. Free tier allows 50,000 reads and writes per month, forever.
+   * [MLab](https://mlab.com/) - mLab is a fully managed cloud database service featuring automated provisioning and scaling of MongoDB databases, backup and recovery, 24/7 monitoring and alerting, web-based management tools, and expert support. Free Sandbox plan provides a single database with 0.5 GB of storage on a shared database server process running on a shared virtual machine (VM).
```

Comment:
this used to be here but for removed when mongodb bought them, since mongodb atlas is already here and provides the same.

If anything, we could add a note on the existing mongodb atlas one that they provide 2 offerings

Reply:
Sorry, I didn't. I think isn't necessary to add, probably the MLab will be discontinued